apps/web/components/apps/App.tsx

```diff
@@ -186,19 +169,28 @@ const Component = ({
                       };
                     }
                     return (
-                      <InstallAppButtonChild appCategories={categories} userAdminTeams={appDbQuery.data?.userAdminTeams} addAppMutationInput={{ type, variant, slug }} multiInstall {...props} />
+                      <Button
+                        StartIcon={Plus}
+                        {...props}
+                        // @TODO: Overriding color and size prevent us from
+                        // having to duplicate InstallAppButton for now.
+                        color="primary"
+                        size="base"
+                        data-testid="install-app-button">
+                        {t("install_another")}
+                      </Button>
```

Comment:
@joeauyeung @hariombalhara Will this have any effect not using the install app button child? Not something im too familiar with